backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/Migrations.java

```diff
@@ -0,0 +1,38 @@
+package org.molgenis.emx2.sql;
+
+import java.io.IOException;
+import org.molgenis.emx2.MolgenisException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Migrations {
+  private static Logger logger = LoggerFactory.getLogger(Migrations.class);
+
+  public static void initOrMigrate(SqlDatabase db) {
+    int version = MetadataUtils.getVersion(db.getJooq());
+
+    // migration steps
+    if (version < 0) MetadataUtils.init(db.getJooq());
+    if (version < 1)
+      executeMigrationFile(db, "migration1.sql", "upgraded MOLGENIS.version_metadata");
+
+    // if cannot migrate then throw a MolgenisException
+
+    // update version, change this number if you create breaking change
+    updateDatabaseVersion(db, 1);
```

Comment:
where are we going to register the link between database-version and release-version ? 
this unnamed int at line 22 seas a bit hidden to me, suggest use a named const

Reply:
I don't think we need this link. And we cannot do that because software version is only known AFTER release, while the database version lives in the code. Added the const, hope that makes it more clear